Makefile

```diff
@@ -394,6 +404,8 @@ service-start-all:
 	docker stack deploy -c docker-stack-as-ap-diag.yml facs-as-ap-diag; \
 	docker stack deploy -c docker-stack-li-ap-energy.yml facs-li-ap-energy; \
 	docker stack deploy -c docker-stack-si-ap-fofb.yml facs-si-ap-fofb; \
+	docker stack deploy -c docker-stack-si-ap-stabinfo.yml facs-si-ap-stabinfo; \
```

Comment:
@anacso17, why is `stabinfo` here as a new entry too? was this a bug?

Reply:
Yes, I noticed this yesterday